src/Inventory/Asset/Processor.php

```diff
@@ -59,7 +59,10 @@ public function prepare(): array
             }
             if (property_exists($val, 'frequency')) {
                 $val->frequency_default = $val->frequency;
-                $val->frequence = $val->frequency;
+                $val->frequency = $val->frequency;
```

Comment:
In database, property is named `frequence`. Are you sure this property has to be renamed ?

Reply:
Field is named `frequence` in `glpi_deviceprocessors` table, but `frequency` in `glpi_items_deviceprocessors`...

Reply:
Anyways, that part of the fix seems indeed wring and should be removed. Missing data is fixed in the else part

Reply:
```suggestion
                $val->frequence = $val->frequency;
```